main/templates/base_home.html

```diff
@@ -5,7 +5,7 @@
     <title>High.Teach</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta charset="utf-8">
-    <link rel="stylesheet" type="text/css" href="{% static 'styles/loggedInPage.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'css/loggedInPage.css' %}">
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"
```

Comment:
Please change the name of the folder of CSS files to match what appears in the static folder or we can decide on one name for the css folders(There are several PRs with different names for the css folder)